src/server/getters/get-market-info.ts

```diff
@@ -52,8 +53,16 @@ export function reshapeMarketsRowToUIMarketInfo(row: MarketsRow, outcomesInfo: A
   return marketInfo;
 }
 
+export function getMarketsWithReportingState(db: Knex, selectColumns?: Array<string>): Knex.QueryBuilder {
+  // TODO: turn leftJoin() into join() once we take care of market_state on market creation
+  const columns = selectColumns ? selectColumns.slice() : ["markets.*"];
+  return db.select(columns.concat("market_state.reportingState"))
+    .from("markets")
+    .leftJoin("market_state", "markets.marketStateID", "market_state.marketStateID");
```

Comment:
Hm, is left join the correct operation here?  This will use all rows from `markets`, even ones that don't have a matching entry in `market_state`.

Reply:
yes, only until we implement market_state creation for new markets. without this, if a row in market_state didn't exist, it wouldn't be returned. The TODO above is so we don't forgot to remove it once we fully implement market_state